decidim-core/app/commands/decidim/invite_user.rb

```diff
@@ -33,7 +33,7 @@ def user
     def update_user
       user.admin = form.role == "admin"
       user.roles << form.role if form.role != "admin"
-      user.roles = user.roles.uniq
+      user.roles = user.roles.uniq.compact
```

Comment:
What if we turn `roles` into a `set`?

Reply:
I don't think postgres supports sets as a column 😕

Reply:
Oh, there's no way to specify attribute coercion in Rails, that's virtus. Ok goodbyeeee